demo/addons/godotopenxr/plugin.cfg

```diff
@@ -0,0 +1,7 @@
+[plugin]
+
+name="GodotOpenXR"
+description="Godot OpenXR loader plugin"
+author="See https://github.com/GodotVR/godot_openxr_loaders/blob/master/CONTRIBUTORS.md"
+version="1.0.0"
```

Comment:
I would change this to `1.1.0` seeing we already release `1.0.0`

Reply:
I'm using this version for testing the maven central dependency given we don't have a `1.1.0` uploaded yet.

Reply:
ahh makes sense :) So we have to remember to update that along with making a release